livre/eq/Channel.cpp

```diff
@@ -275,16 +275,9 @@ class Channel
         _renderViewPtr->setViewport( viewport );
 
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
```

Comment:
Previously generate rendering set was doing the selection of visibles as well ( which may be overwritten by commit ). Therefore I was doing the second time. Since this is separated by sort last rendering changes it may be useless. But should be removed with care. As I have said the async/sync/animation modes have to be carefully tested.